Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: avoid lalrpop trawling the current working directory

See https://github.com/lalrpop/lalrpop/issues/908 for more discussion

diff --git a/build.rs b/build.rs
index b8977c5..c1c571c 100644
--- a/build.rs
+++ b/build.rs
@@ -4,7 +4,7 @@ use std::io::{self, Write};
 use std::path::PathBuf;
 
 fn main() {
-    lalrpop::process_root().unwrap();
+    lalrpop::Configuration::new().use_cargo_dir_conventions().process().unwrap();
     include_test_data().unwrap();
 }
 
